common/annotations/en.xml

```diff
@@ -736,6 +736,11 @@ annotations.
 		<annotation cp="😂" type="tts">face with tears of joy</annotation>
 		<annotation cp="🙂">face | slightly smiling face | smile</annotation>
 		<annotation cp="🙂" type="tts">slightly smiling face</annotation>
+		<annotation cp="🙂‍↕">head shaking vertically | nod, yes</annotation> <!-- 1F642 200D 2195 -->
+		<annotation cp="🙂‍↕" type="tts">head shaking vertically</annotation>
+		<!-- Generated lines from Emoji Data v15.1, using GenerateCldrData.java -->
```

Comment:
This comment "Generated lines from Emoji Data v15.1, using GenerateCldrData.java" is no longer associated with a specific set of lines due to the re-ordering; doubtful whether it makes sense to keep the comment in the data; not a big deal

Reply:
Yes, I will fix that in another PR, thanks